x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/EvalOperator.java

```diff
@@ -51,5 +51,15 @@ public interface ExpressionEvaluator {
         Block eval(Page page);
     }
 
-    public static final ExpressionEvaluator CONSTANT_NULL = page -> Block.constantNullBlock(page.getPositionCount());
+    public static final ExpressionEvaluator CONSTANT_NULL = new ExpressionEvaluator() {
+        @Override
+        public Block eval(Page page) {
+            return Block.constantNullBlock(page.getPositionCount());
+        }
+
+        @Override
+        public String toString() {
+            return "ConstantNull";
+        }
+    };
```

Comment:
This `toString` shows up in explain output so it's nice not to get a random closure.